turbinia/lib/docker_manager.py

```diff
@@ -42,6 +43,39 @@ def IsBlockDevice(path):
   return stat.S_ISBLK(mode)
 
 
+def GetDockerPath(mount_path):
+  """Retrieves the Docker installation path.
+
+  Args:
+    mount_path(str): The mount path of the Evidence object.
+
+  Returns:
+    docker_path(str): The Docker installation path.
+  """
+  docker_path = None
+  etc_path = os.path.join(mount_path, 'etc/docker/daemon.json')
+  if os.path.exists(mount_path):
+    try:
+      with open(etc_path) as etc_handle:
+        json_obj = json.loads(etc_handle.read())['data-root']
+        # Remove starting / so paths can join
+        if json_obj.startswith('/'):
+          json_obj = json_obj[1:]
+        docker_path = os.path.join(mount_path, json_obj)
+    except Exception as exception:
+      log.error(
+          'Error parsing the Docker daemon config file due to: {0:s}. '
+          'Using default Docker installation path'.format(str(exception)))
+
+  # If file not found or error occured parsing Docker config file.
+  if docker_path is None:
+    log.warning(
+        'Docker daemon confile file not found. '
+        'Using default Docker installation path.')
+    docker_path = os.path.join(mount_path, 'var/lib/docker')
```

Comment:
I wonder if we should test if this exists, and throw an exception if it doesn't?  Or is this checked by callers of this method?

Reply:
It is checked by the callers of the method (in the run task of the code).